script.js

```diff
@@ -0,0 +1,32 @@
+
+function addTask() {
+  const description = addInput.value;
+  if (description) {
+    Task.addTask(description);
+    addInput.value = '';
+    renderTasks();
+  }
+}
+
+renderTasks();
+
+addButton.addEventListener('click', () => {
+  addTask();
+});
+
+addInput.addEventListener('keypress', (event) => {
+  if (event.key === 'Enter') {
+    addTask();
+  }
```

Comment:
Based on the provided code, here are some observations and suggestions for improvement:

JavaScript Best Practices:

Function Naming: The code includes three identical addTask() functions. It is recommended to have unique and descriptive names for each function to avoid confusion.
Variable Declaration: The const keyword is correctly used to declare variables in most cases. However, it seems that addInput, Task, addButton, addInput, and clearAllButton are not defined in the code snippet. Make sure these variables are properly defined and accessible.